DirectProgramming/DPC++/ParallelPatterns/openmp_reduction/sample.json

```diff
@@ -0,0 +1,29 @@
+{
+  "guid": "3C2AA983-1B31-4C05-BCA0-724F159B8535", 
+  "name": "openmp_reduction",
+  "categories": ["Toolkit/Intel® oneAPI Base Toolkit/oneAPI DPC++ Compiler/oneAPI DPC++ Library/CPU and GPU"],
+  "description": "It models transform reduce in different ways showing capability of oneAPI.",
+  "toolchain": ["icpx"],
+  "languages": [{"cpp":{}}],
+  "targetDevice": ["CPU", "GPU"],
+  "os": ["linux"],
```

Comment:
Missing "ciTests" field in the sample.json.  "ciTests": Please look at simple-add example for reference